components/Events/Filters.js

```diff
@@ -0,0 +1,142 @@
+import {faChevronUp, faChevronDown} from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useState, React } from 'react';
+import events from '../../data/event';
+import styles from '../../styles/components/Events/Filters.module.scss';
+
+
+export default function filters(props) {
+	const days = ['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday'];
+	const [filterOpen, setFilterOpen] = useState(false);
+    const [searchValue,setSearchValue] = useState('');
+	const [chosenDays, setChosenDays] = useState(new Array(7).fill(false));
+    const [chosenComms, setChosenComms] = useState({
+        studio: false,
+        icpc: false,
+        design: false,
+        cyber: false,
+        teachLA: false,
+        w: false,
+        ai: false,
+        hack: false,
+        board: false,
+    });
+    const [fromTime, setFromTime] = useState('');
+    const [toTime, setToTime] = useState('');
+    const [chosenLoc, setChosenLoc] = useState({
+        online: false,
+        in_person: false,
+    });
+
+    const handleSearch = () => {
+        let allEvents = [...events];
+        //Check search input
+        const lowerCaseSearch = searchValue.toLowerCase();
+        allEvents = allEvents.filter(item => {
+            if (String(item.title).toLowerCase().includes(lowerCaseSearch)) return true;
+            if (String(item.committee).toLowerCase().includes(lowerCaseSearch)) return true;
+            if (String(item.location).toLowerCase().includes(lowerCaseSearch)) return true;
+            if (String(item.description).toLowerCase().includes(lowerCaseSearch)) return true;
+            return false;
+        });
+        //Check filters
+        const timeEmpty = fromTime === '' || toTime === '';
+        const commEmpty = Object.keys(chosenComms).every(comm => !chosenComms[comm]);
+        const daysEmpty = chosenDays.every(day => !day);
+        const locEmpty =  Object.keys(chosenLoc).every(loc => !chosenLoc[loc]);
```

Comment:
nit: prefer to start boolean values with `is`

Took me a bit to realize what these variables meant